Source/WebKit/Platform/IPC/StreamClientConnection.cpp

```diff
@@ -59,11 +59,11 @@ void StreamClientConnection::DedicatedConnectionClient::didReceiveInvalidMessage
 StreamClientConnection::StreamConnectionPair StreamClientConnection::create(unsigned bufferSizeLog2)
```

Comment:
@kkinnunen-apple seems like this should be `std::optional` as well. Commented out the below lines because it required the default constructor of `SharedMemory::Handle`.